components/card/stories/card.mdx

```diff
@@ -0,0 +1,62 @@
+import { ArgTypes, Canvas, Meta, Description, Title } from '@storybook/blocks';
+
+import * as CardStories from './card.stories';
+
+<Meta of={CardStories} title="Docs" />
+
+<Title of={CardStories} />
+<Description of={CardStories} />
+
+## Standard
+
+<Canvas of={CardStories.Default} />
+
+### No image
+
+<Canvas of={CardStories.NoImage} />
+
+### Focused
+
+<Canvas of={CardStories.Focused} />
+
+### Selected
+
+<Canvas of={CardStories.Selected} />
+
+## Quiet
+
+<Canvas of={CardStories.Quiet} />
+
+### File
```

Comment:
This and the several variants below it could use "Quiet" in the heading so as not to be confused with their non-quiet counterparts.